docs/index.rst

```diff
@@ -28,4 +28,6 @@ Welcome to pysat's documentation!
 .. admonition:: DISTRIBUTION STATEMENT A: Approved for public release.
 		Distribution is unlimited.
 
-    This work was supported by the Office of Naval Research.
+Funding
+=======
+.. include:: ../acknowledgements.txt
```

Comment:
Recommend including funding in the main doc, perhaps within "citing.rst" or as it's own section.